### PR TITLE
For upstream

### PR DESCRIPTION
**Pull request by AlexIljin**
Please consider this contribution of assorted fixes.

### CODE REVIEW

**Comment by AlexIljin** [comment]
No time to review, huh?

**Comment by mrjbq7** [comment]
Traveling today.  Will look at it when I get time.

File: vm/os-windows.cpp [hunk 1]
```diff
@@ -151,6 +151,12 @@ uint64_t nano_count() {
   static uint32_t hi = 0;
   static uint32_t lo = 0;
 
+  // Note: on older systems QueryPerformanceCounter may be unreliable
+  // until you add /usepmtimer to Boot.ini. I had an issue where two
+  // nano_count calls would show a difference of about 1 second,
+  // while actually about 80 seconds have passed. The /usepmtimer
+  // switch cured the issue on that PC (WinXP Pro SP3 32-bit).
+  // See also http://www.virtualdub.org/blog/pivot/entry.php?id=106
   LARGE_INTEGER count;
```
Comment (bjourne) [inline vm/os-windows.cpp]: It's no biggie, but comments should be with `/* */`. See #1651

**Comment by bjourne** [comment]
I've reviewed and it looks fine to me!

**Comment by mrjbq7** [comment]
Merged, thanks!

**Pull request closed without merging**